docs/develop/how_to_dev_backend_en.md

```diff
@@ -1,4 +1,4 @@
-# VisualDL Backend / SDK structure and Develop Guide
+# Backend Architecture
```

Comment:
Maybe just call Backend Guide? We have a Frontend Guide already. Also this doc talks about how to dev and architecture is a sub session.

Reply:
`Backend Guide` works for me too.